docs/tools/mbed_targets.md

```diff
@@ -253,6 +257,10 @@ The [`"device_name"`](/docs/development/porting/index.html) attribute it `target
 
 The `device_name` key in `targets.json` is `MK20DX256xxx7` for any target that uses this particular MCU.
 
+#### `detect_code`
+
+The `detect_code` contains four ASCII characters containing only hexadecimal values (A-F and 0-9). This code is the same for all platforms of the same type. For example, all K64F platforms have a detect_code of 0240. mbedls uses this to identify the platform. This field is not used anymore. Mbed-ls has its own database of detect codes: https://github.com/ARMmbed/mbed-ls/blob/master/mbed_lstools/platform_database.py#L45
```

Comment:
We should probably refer to "Mbed LS" in a consistent fashion. I would stick with the "Mbed LS" styling.